src/assets/i18n/de_DE.json

```
@@ -923,7 +923,7 @@
   "product.items.sorting.default.label": "Standardsortierung",
   "product.label.new.text": "NEU",
   "product.label.sale.text": "SALE",
-  "product.label.topseller.text": "TOP",
+  "product.label.topseller.text": "Beliebt",
```

```suggestion
  "product.label.topseller.text": "TOP",
```

Please leave it as it is.